packages/plugin-team/src/decrypter.js

```diff
@@ -0,0 +1,64 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import {base64} from '@ciscospark/common';
+import {Decrypter} from '@ciscospark/plugin-conversation';
+
+/**
+ * Decrypts a team
+ * @param {Object} team
+ * @param {Encryption~Key|string} key
+ * @param {Object} options
+ * @param {Object} parentObject
+ * @returns {Promise} Resolves with the decrypted team
+ * @private
+ */
+Decrypter.prototype._decryptTeam = function _decryptTeam(team, key, options, parentObject) {
+  let promises = [];
+  if (team.conversations.items) {
+    promises = team.conversations.items.map((item) => this.decryptObject(item, options));
+  }
+
+  const usableKey = key || team.encryptionKeyUrl;
+
+  if (usableKey && team.displayName) {
+    try {
```

Comment:
The original conversation [decrypter](https://github.com/ciscospark/spark-js-sdk/blob/master/src/client/services/conversation/decrypter.js#L164-L192) was really defensive, so the teams one was as well. Will change this to be more like the new and improved plugin-conversation encrypter.